build_tools/travis/after_wheel_build.sh

```diff
@@ -9,3 +9,11 @@ else
     echo "Build artifacts were not created. Skipping deployment..."
     exit 1
 fi
+
+# If we get to this point, the build worked, and we need to check for README rendering
```

Comment:
@tgsmith61591 I added this check to Travis too, but it will only run when we are deploying. I don't have a good way to test it, so if we want to pull it out for this PR, that's fine by me